src/components/Card.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import imagePin from "../images/pin.svg";
+
+export default function Card(props) {
+    return (
+        <div className="card">
+            <img src={props.imageUrl}/>
+            <div className="card--text">
+                <div className="card--text--top">
+                    <h3><img src={imagePin}/>{props.location}</h3>

```

Comment:
Brak alta.

H3, ale nie mamy nigdzie H1 w tej strukturze. Strzelam, że można to zastąpić <strong>